### PR TITLE
Change 6 to six

### PR DESCRIPTION
**Pull request by alsargent**

### CODE REVIEW

File: docs/selenium-javascript.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ sidebarDepth: 1
 ---
 # Selenium JavaScript
 
-This quickstart has 6 parts with lot of gifs and pictures to help you quickly get upto speed and know how powerful Applitools' AI really is. Not that it will take about 20 minutes after you install the code. 
+This quickstart has six parts with lot of gifs and pictures to help you quickly get upto speed and know how powerful Applitools' AI really is. Not that it will take about 20 minutes after you install the code.
```
Comment (rajaraodv) [inline docs/selenium-javascript.md:6]: Hey, I like 6 as a number

Comment (alsargent) [reply docs/selenium-javascript.md:6]: Yeah, but Strunk & White...

Comment (rajaraodv) [reply docs/selenium-javascript.md:6]: Okie dokie

File: docs/selenium-javascript.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ sidebarDepth: 1
 ---
 # Selenium JavaScript
 
-This quickstart has 6 parts with lot of gifs and pictures to help you quickly get upto speed and know how powerful Applitools' AI really is. Not that it will take about 20 minutes after you install the code.
```
Comment (rajaraodv) [inline docs/selenium-javascript.md:6]: bla

**Review by rajaraodv — CHANGES_REQUESTED** [review]

**Pull request closed without merging**